core/modules/image/tests/image.test

```diff
@@ -246,7 +258,23 @@ class ImageStylesPathAndUrlUnitTest extends BackdropWebTestCase {
     $this->assertRaw(file_get_contents($generated_uri), 'URL returns expected file.');
```

Comment:
This line already verifies that the returned response matches the generated image contents.